### ISSUE TITLE
If Else statements in ride are displayed on one long line after a reload or ride restart

### ISSUE THREAD

**Issue by moliveras** [robotframework/RIDE#2023]
System info:
Centos 7
python 2.7.5
robotframework 3.1.2
ride 1.7.4b2

Create a IF ELSE statement as follows:

![image](https://user-images.githubusercontent.com/38561769/66160096-6246ca00-e5f7-11e9-992c-7c931fa155d7.png)

```
IF Format Test
    ${var}    Set Variable    kiwi
    Run Keyword If    '${var}' == 'apple'    Log    It's a red fruit
    ...    ELSE IF    '${var}' == 'banana'    Log    It's a yellow fruit
    ...    ELSE    Log    Its not an apple or a banana
    [Teardown]
```
After I reload the test under the File menu or restart ride, the ride UI displays the entire Keyword  on one line.
![image](https://user-images.githubusercontent.com/38561769/66160187-991ce000-e5f7-11e9-91cd-8c11d22c6865.png)

Everything works as expected, however it is much easier to read when displayed on multiple lines, especially if there are several conditions.  Would it be possible to preserve the display in ride using multiple lines?

BTW - This version This is a huge improvement over 1.5.2.1!

**Comment by HelioGuilherme66** [comment]
@moliveras The parsing of files is done using robot core libraries (in this case, `tidy`). That problem will disappear with changes in that code. Please see this [issue](https://github.com/robotframework/robotframework/issues/2579) at Robot Framework project.

**Comment by HelioGuilherme66** [comment]
This is now fixed on v2.0.8dev5.
